api_app/service_bus/substitutions.py

```diff
@@ -89,18 +103,54 @@ def substitute_value(val: str, primary_resource_dict: dict) -> Union[dict, list,
             t = p[0:p.index("}}")]
             tokens.append(t)
 
+    dict_to_use = None
     for t in tokens:
-        # t = "resource.properties.prop_1"
+        # t = "{resource[.parent][.parent].properties.prop_1"
```

Comment:
why is this commented out?

Reply:
The comment says this is an example of how prop substitute looks like.
I added that 2 '.parent' are also an option